data-model/fdo-type/term/term.json

```diff
@@ -0,0 +1,195 @@
+      ]
+    },
+    "ods:status": {
+      "type": "string",
+      "enum": [
+        "ods:Draft",
+        "ods:Active",
+        "ods:Tombstone"
+      ],
+      "description": "The status of the Digital Object",
+      "examples": [
+        "ods:Active"
+      ]
+    },
+    "schema:version": {
+      "type": "integer",
+      "minimum": 1,
+      "description": "Version of the Schema, https://schema.org/version",
+      "examples": [
+        1
+      ]
+    },
+    "schema:name": {
+      "type": "string",
+      "description": "Name of the Term, https://schema.org/name",
+      "examples": [
+        "ods:DigitalSpecimen"
+      ]
+    },
+    "skos:prefLabel": {
+      "type": "string",
+      "description": "The preferred lexical label for the term",
+      "examples": [
+        "Digital Specimen"
+      ]
+    },
+    "schema:dateCreated": {
+      "type": "string",
+      "description": "Timestamp of creation. Internally generated, https://schema.org/dateCreated",
+      "format": "date-time",
+      "examples": [
+        "2021-06-01T12:00:000Z"
+      ]
+    },
+    "schema:dateModified": {
+      "type": "string",
+      "description": "Timestamp of last modification. Internally generated, https://schema.org/dateModified",
+      "format": "date-time",
+      "examples": [
+        "2021-06-06T13:22:00.445Z"
+      ]
+    },
+    "schema:creator": {
+      "type": "object",
+      "description": "Contains an ods:Agent object",
+      "$ref": "https://schemas.dissco.tech/schemas/fdo-type/shared-model/0.3.0/agent.json",
+      "examples": [
+        {
+          "@id": "https://orcid.org/0000-0002-1825-0097",
+          "@type": "schema:Person",
+          "schema:name": "John Doe",
+          "schema:email": "john.doe@example.com"
+        }
+      ]
+    },
+     "schema:contributor": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "List of contributors",
+      "examples": [
+        ["Jane Smith", "Bob Johnson"]
+      ]
+    },
+    "skos:definition": {
+      "type": "string",
+      "description": "A definition of what the term represents.",
+      "examples": [
+        "A digital representation of a physical specimen"
+      ]
+    },
+   
+    "dcterms:licence": {
+      "type": "string",
+      "format": "uri",
+      "description": "License URI",
+      "enum": [
+        "https://creativecommons.org/licenses/by/4.0/",
+        "https://creativecommons.org/licenses/by-sa/4.0/",
+        "https://creativecommons.org/licenses/by-nc/4.0/",
+        "https://creativecommons.org/licenses/by-nc-sa/4.0/"
+      ],
+      "examples": [
+        "https://creativecommons.org/licenses/by/4.0/"
+      ]
+    },
+    "ods:belongsToClass": {
+      "type": "string",
+      "format": "uri",
+      "description": "URI of the class this term belongs to",
+      "examples": [
+        "https://schema.org/Thing"
+      ]
+    },
+    "ods:repeatable": {
```

Comment:
This doesn't comply to our `ods` naming convention.
For booleans this is `ods:isXxx`
So in this case it should be `ods:isRepeatable` in line with `ods:isRequired`